### ISSUE TITLE
Bank Loans - Java JDBC MySQL 

### ISSUE THREAD

**Issue by lespabrandzulic82** [alptium/branko.spalovic#14]

**Comment by lespabrandzulic82** [comment]
@valentinacupac  please see my solution for this task with database...thanks.

**Comment by valentinajemuovic** [comment]
@lespabrandzulic82, I opened up your code here https://github.com/alptium/branko.spalovic/tree/master/bank-loans/src/bankloans but I cannot see the code for the databases...

**Comment by lespabrandzulic82** [comment]
i have committed, but there is no changes on git hub, i don't understand ... @valentinacupac

**Comment by valentinajemuovic** [comment]
@lespabrandzulic82, did you push your code?

**Comment by lespabrandzulic82** [comment]
@valentinacupac yes, i did. But i can't pull it on github, like nothing is happening...

**Comment by lespabrandzulic82** [comment]
@valentinacupac I succeed to commit, could you take a look now? thanks.
